cloudsync_onedrive.py

```diff
@@ -1223,11 +1248,15 @@ def _info_item(self, item, path=None) -> OneDriveInfo:
 
         if path is None:
             path = odi.path
+        path_orig = path
+        if self._is_biz and self.namespace.shared_folder_path:
+            # make path relative to the shared folder
+            path = self.is_subpath(self.namespace.shared_folder_path, path) or path
```

Comment:
This code is a duplicate, consider creating a method that takes orig_path, and returns a (possibly modified) path

Reply:
done